packages/ui/src/loading-indicator/loading-indicator.ts

```diff
@@ -9,19 +9,19 @@ const DEFAULT_STYLES: LoadingIndicatorStyles = {
 const ROTATION_ANIMATION = 'embedded-checkout-loading-indicator-rotation';
 
 export default class LoadingIndicator {
-    private _container: HTMLElement;
-    private _indicator: HTMLElement;
-    private _styles: LoadingIndicatorStyles;
+    private container: HTMLElement;
+    private indicator: HTMLElement;
+    private styles: LoadingIndicatorStyles;
```

Comment:
LGTM. Is this name change intentional?

Reply:
Yes. This changes are related to `naming-convention` eslint rule what should be removed in the future.

Eslint rule in ui package: https://github.com/bigcommerce/checkout-sdk-js/blob/master/packages/ui/.eslintrc.json#L8